### PR TITLE
Hardening/117 delete individual context entity attribute

### PR DESCRIPTION
**Pull request by kzangeli**
Another convop ...
GET /v1/contextEntities/{EID}/attributes/{attrName}/{metaId}

### CODE REVIEW

**Comment by kzangeli** [comment]
The name of this branch ...
I think I might have forgotten to create a new branch, and just continued on the branch from the previous PR.
Doesn't really matter, I think we will just see two PRs with branches with the same name.

File: test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test [hunk 1]
```diff
@@ -155,7 +155,7 @@ Date: REGEX(.*)
   <statusCode>
     <code>404</code>
     <reasonPhrase>No context element found</reasonPhrase>
-    <details>Entity-Attribute pair: /E01-A1/</details>
+    <details>Entity id: /E01/</details>
```
Comment (fgalan) [inline test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test]: We have "lost" some info... previously details include entity-attribute pair, now only entity. Do we want this in this way?

File: test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+--SHELL--
+
+#
+# 01. GET /v1/contextEntities/E1/attributes/A1/ID_A?entity::type=T1 and see it fail
+# 02. Create E1/T1/A1 with metadata ID ID_A
+# 03. GET /v1/contextEntities/E1/attributes/A1/ID_A?entity::type=T1 and see it work
+# 04. GET /v1/contextEntities/E1/attributes/A1/ID_B?entity::type=T1 and see it fail
+# 05. GET /v1/contextEntities/E1/attributes/A1/ID_A?entity::type=T1&attributesFormat=object
+# 06. Create E1//A1 with metadata ID ID_B
+# 07. GET /v1/contextEntities/E1/attributes/A1/ID_B?!exist=entity::type and see it work
```
Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test]: Looking to the list of supported arguments in getAttributeValueInstance() preamble, it seems that `?exist=entity::type` is not covered by these steps.

File: test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test (hunk 1 above)
Comment (kzangeli) [inline test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test]: I know.
This is how it came out.
I didn't care to spend time to see why, for this request that we don't even want ... :-)

If you find it important, I'll try to fix it to return the same response as before.

File: test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test (hunk 2 above)
Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test]: True.  I'll add a part that checks that uri param

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test]: Fixed in d12396d

File: test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test (hunk 1 above)
Comment (fgalan) [inline test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test]: Good point. Along with backward compatibility is preserved (as in this case), let's deal with error message improvements in the future (in the case they are needed at the end).

NTC

File: test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+        "code": "404", 
+        "details": "Entity id: /E2/", 
+        "reasonPhrase": "No context element found"
+    }
+}
+
+
+--TEARDOWN--
+brokerStop CB
+#dbDrop CB
```
Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test]: Should be uncommented?

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test]: Yes, of course

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test]: Fixed in cd9ad55

**Comment by fgalan** [comment]
LGTM

**Pull request merged**